distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -1,94 +1,107 @@
 === Bug fixes
 
-This update introduces the following bug fixes for the {JaegerShortName}:
+This update introduces the following bug fixes for the {TempoShortName}:
 
-* Fixed support for disconnected environments when using the `oc adm catalog mirror` CLI command. (link:https://issues.redhat.com/browse/TRACING-3546[TRACING-3546])
+* Fixed operator RBAC for the monitor tab when the operator is deployed in arbitrary namespace. (link:https://issues.redhat.com/browse/TRACING-3786[TRACING-3786])
+* Fixed support for IPv6 clusters.(link:https://issues.redhat.com/browse/TRACING-3226[TRACING-3226])
 
-[id="known-issues_jaeger-release-notes_distributed-tracing-rn-3-0"]
+[id="distributed-tracing-rn_3-1_tempo-release-notes_known-issues"]
 === Known issues
 
-* Currently, Apache Spark is not supported.
+//There is currently a known issue:
+There are currently known issues:
 
-ifndef::openshift-rosa[]
+* Currently, when used with the {TempoOperator}, the Jaeger UI only displays services that have sent traces in the last 15 minutes. For services that did not send traces in the last 15 minutes, traces are still stored but not displayed in the Jaeger UI. (link:https://issues.redhat.com/browse/TRACING-3139[TRACING-3139])
+* Currently, the {TempoShortName} fails on the IBM Z (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
 
-* Currently, the streaming deployment via AMQ/Kafka is not supported on the IBM Z and IBM Power Systems architectures.
-endif::openshift-rosa[]
+// Jaeger section
+[id="distributed-tracing-rn_3-1_jaeger-release-notes"]
+== {JaegerName}
 
-// Tempo section
-[id="tempo-release-notes_distributed-tracing-rn-3-0"]
-== {TempoName}
+[id="distributed-tracing-rn_3-1_jaeger-release-notes_support-for-elasticsearch-operator"]
+=== Support for Elasticsearch Operator
 
-[id="new-features-and-enhancements_tempo-release-notes_distributed-tracing-rn-3-0"]
-=== New features and enhancements
+{JaegerName} 3.1 is supported for use with the OpenShift Elasticsearch Operator 5.6, 5.7, and 5.8.
 
-This update introduces the following enhancements for the {TempoShortName}:
+[id="distributed-tracing-rn_3-1_jaeger-release-notes_deprecated-functionality"]
+=== Deprecated functionality
+
+In Red Hat OpenShift distributed tracing 3.1, Jaeger and Elasticsearch are deprecated, and both are planned to be removed in a future release. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements.
+
+In Red Hat OpenShift distributed tracing 3.1, Tempo provided by the {TempoOperator} and the OpenTelemetry collector provided by the Red Hat build of OpenTelemetry are the preferred Operators for distributed tracing collection and storage. The OpenTelemetry and Tempo distributed tracing stack is to be adopted by all users because this will be the stack that will be enhanced going forward.
```

Comment:
Use the attribute {OTELName} for "Red Hat build of OpenTelemetry."

See: https://redhat-documentation.github.io/supplementary-style-guide/#_product_names_and_version_references

Reply:
Thank you so much. Grepped for and fixed.